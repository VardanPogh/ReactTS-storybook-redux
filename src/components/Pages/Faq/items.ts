const accordionItems = [
  {
    title: 'Test 1',
    content:
      'Lorem ipsum dolor sit amet consectetur adipisicing elit. Consequatur excepturi blanditiis repudiandae fuga sit libero? Recusandae, quasi incidunt labore animi magni ducimus id vitae distinctio tempore nesciunt repudiandae quia! Qui?',
  },
  {
    title: 'Test 2',
    content:
      'Lorem ipsum dolor sit amet consectetur adipisicing elit. Excepturi, voluptatem ea fugiat quia in ad accusantium asperiores, blanditiis molestias omnis reprehenderit a, incidunt dolorum quisquam dolor doloremque debitis perferendis maiores.',
  },
  {
    title: 'Test 3',
    content:
      'Lorem ipsum dolor sit amet consectetur adipisicing elit. Quas nemo dolorum error quo exercitationem aliquam aperiam itaque ex qui deserunt consequatur recusandae est deleniti aliquid, velit ipsam tempore voluptas necessitatibus?',
  },
]

export default accordionItems
