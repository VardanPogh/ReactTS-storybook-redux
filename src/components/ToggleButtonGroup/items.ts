const items = [
  {
    value: 'check',
    img: 'img/blue-1.svg',
    description:
      'Lorem ipsum dolor sit amet consectetur adipisicing elit. Placeat.',
  },
  {
    value: 'check1',
    img: 'img/blue-2.svg',
    description:
      'Lorem ipsum dolor sit amet consectetur adipisicing elit. Placeat.',
  },
  {
    value: 'check2',
    img: 'img/blue-3.svg',
    description:
      'Lorem ipsum dolor sit amet consectetur adipisicing elit. Placeat.',
  },
  {
    value: 'check3',
    img: 'img/blue-4.svg',
    description:
      'Lorem ipsum dolor sit amet consectetur adipisicing elit. Placeat.',
  },
  {
    value: 'check4',
    img: 'img/blue-5.svg',
    description:
      'Lorem ipsum dolor sit amet consectetur adipisicing elit. Placeat.',
  },
]

export default items
