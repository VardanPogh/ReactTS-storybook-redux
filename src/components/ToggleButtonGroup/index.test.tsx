import React from 'react'
import { render } from '@testing-library/react'

import ToggleGroup from '.'
import items from './items'

describe('ToggleGroup Test', () => {
  test('Load ToggleGroup', () => {
    const myMock = jest.fn()
    const { container } = render(
      <ToggleGroup items={items} onChange={myMock} />
    )
    expect(container).toMatchSnapshot()
  })
})
