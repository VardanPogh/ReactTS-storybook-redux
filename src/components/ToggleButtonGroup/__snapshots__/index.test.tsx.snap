// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ToggleGroup Test Load ToggleGroup 1`] = `
<div>
  <div
    class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3"
  >
    <div
      class="MuiGrid-root makeStyles-items-1 MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-true"
    >
      <button
        aria-pressed="false"
        class="MuiButtonBase-root MuiToggleButton-root makeStyles-toggle-3"
        tabindex="0"
        type="button"
        value="check"
      >
        <span
          class="MuiToggleButton-label"
        >
          <div
            class="makeStyles-innerContainer-2"
          >
            <img
              alt="toggle-choice"
              src="img/blue-1.svg"
            />
            <p>
              Lorem ipsum dolor sit amet consectetur adipisicing elit. Placeat.
            </p>
          </div>
        </span>
        <span
          class="MuiTouchRipple-root"
        />
      </button>
    </div>
    <div
      class="MuiGrid-root makeStyles-items-1 MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-true"
    >
      <button
        aria-pressed="false"
        class="MuiButtonBase-root MuiToggleButton-root makeStyles-toggle-3"
        tabindex="0"
        type="button"
        value="check1"
      >
        <span
          class="MuiToggleButton-label"
        >
          <div
            class="makeStyles-innerContainer-2"
          >
            <img
              alt="toggle-choice"
              src="img/blue-2.svg"
            />
            <p>
              Lorem ipsum dolor sit amet consectetur adipisicing elit. Placeat.
            </p>
          </div>
        </span>
        <span
          class="MuiTouchRipple-root"
        />
      </button>
    </div>
    <div
      class="MuiGrid-root makeStyles-items-1 MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-true"
    >
      <button
        aria-pressed="false"
        class="MuiButtonBase-root MuiToggleButton-root makeStyles-toggle-3"
        tabindex="0"
        type="button"
        value="check2"
      >
        <span
          class="MuiToggleButton-label"
        >
          <div
            class="makeStyles-innerContainer-2"
          >
            <img
              alt="toggle-choice"
              src="img/blue-3.svg"
            />
            <p>
              Lorem ipsum dolor sit amet consectetur adipisicing elit. Placeat.
            </p>
          </div>
        </span>
        <span
          class="MuiTouchRipple-root"
        />
      </button>
    </div>
    <div
      class="MuiGrid-root makeStyles-items-1 MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-true"
    >
      <button
        aria-pressed="false"
        class="MuiButtonBase-root MuiToggleButton-root makeStyles-toggle-3"
        tabindex="0"
        type="button"
        value="check3"
      >
        <span
          class="MuiToggleButton-label"
        >
          <div
            class="makeStyles-innerContainer-2"
          >
            <img
              alt="toggle-choice"
              src="img/blue-4.svg"
            />
            <p>
              Lorem ipsum dolor sit amet consectetur adipisicing elit. Placeat.
            </p>
          </div>
        </span>
        <span
          class="MuiTouchRipple-root"
        />
      </button>
    </div>
    <div
      class="MuiGrid-root makeStyles-items-1 MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-true"
    >
      <button
        aria-pressed="false"
        class="MuiButtonBase-root MuiToggleButton-root makeStyles-toggle-3"
        tabindex="0"
        type="button"
        value="check4"
      >
        <span
          class="MuiToggleButton-label"
        >
          <div
            class="makeStyles-innerContainer-2"
          >
            <img
              alt="toggle-choice"
              src="img/blue-5.svg"
            />
            <p>
              Lorem ipsum dolor sit amet consectetur adipisicing elit. Placeat.
            </p>
          </div>
        </span>
        <span
          class="MuiTouchRipple-root"
        />
      </button>
    </div>
  </div>
</div>
`;
